src/dswx_sar/schemas/dswx_s1.yaml

```
@@ -41,6 +41,9 @@ runconfig:
             # NOAA GSHHS shapefile description
             shoreline_shapefile_description: str(required=False)
 
+            # algorithm parameter 
+            algorithm_parameter: str(required=True)
```

we usually have more than one parameter.
```suggestion
            algorithm_parameters: str(required=True)
```